tests/foreman/ui/test_domain.py

```diff
@@ -95,6 +96,10 @@ def test_update_domain(self, testdata):
         @Assert: Domain is updated
 
         """
+        bug_id = testdata.pop('bugzilla', None)
+        if bug_id is not None and bz_bug_is_open(bug_id):
+            self.skipTest('Bugzilla bug {0} is open.'.format(bug_id))
```

Comment:
It will just skip the specific data where bz-id is included. For example here we blocked the html data, though `skip_if_bug_open` skips the complete test.